app-gateway.tf

```diff
@@ -8,6 +8,16 @@ data "azurerm_key_vault_secret" "cert" {
   key_vault_id = "${data.azurerm_key_vault.infra_vault.id}"
 }
 
+data "azurerm_key_vault_secret" "exela_ip" {
```

Comment:
let's call this as scanning_supplier_ip rather than disclosing supplier name in open source